printrun/projectlayer.py

```diff
@@ -485,7 +522,7 @@ def set_estimated_time(self):
         if not hasattr(self, 'layers'):
             return
 
-        current_layer = int(self.current_layer.GetLabel())
+        current_layer = int(float(self.current_layer.GetLabel()))
```

Comment:
int() throws a ValueError if the string contains a float (like "1.0" and not "1"). So it was added as a safety measure and I didn't think much about it.
Since it's never been an issue before, I probably shouldn't change it... 😅